ols/src/query_helpers/docs_summarizer.py

```diff
@@ -96,7 +97,12 @@ def summarize(
             logger.warning("Proceeding without RAG content. Check start up messages.")
 
         rag_context = "\n\n".join(rag_context_data.get("text", []))
-        referenced_documents = rag_context_data.get("docs_url", [])
+        referenced_documents = [
+            ReferencedDocument(docs_url=docs_url, title=title)
+            for docs_url, title in zip(
+                rag_context_data.get("docs_url", []), rag_context_data.get("title", [])
```

Comment:
do we not need an updated RAG image for this metadata to be present?

Reply:
which is now done